dependencies.xml

```diff
@@ -59,7 +59,8 @@ Note all install methods after "main" take
     <nomkl os='linux' skip_check='True'/>
     <numexpr os='linux'/>
     <cmake skip_check='True' optional='True'/>
-    <ray os="mac,linux" source="pip" pip_extra="[default]">1.9</ray>
+    <ray source="pip" pip_extra="[default]">1.12</ray>
+    <redis source="pip" os='windows'/>
```

Comment:
For some reason, redis is needed by ray but in windows pip does not get it as a dependency.